main.tf

```diff
@@ -55,11 +55,12 @@ resource "google_storage_bucket" "main" {
   storage_class      = "REGIONAL"
   labels             = var.bucket_labels
   bucket_policy_only = true
+  count              = var.create_bucket == true ? 1 : 0
```

Comment:
nit: can we keep count towards the top of the resource, for more readability.
[Example](https://github.com/terraform-google-modules/terraform-google-kms/blob/3f01a09f816da0b39f1ab35bed8f6cea694bed57/main.tf#L28)

Reply:
adopted